release-process.md

```diff
@@ -75,9 +44,37 @@ For example if you are cutting RC for 1.0.2, mark such issues as `FIXED` in 1.0.
 release, and change them to the current release.
```

Comment:
not sure if it's important, in JIRA we are using Resolve Issue, Fixed Version/s, Target Version/s